# Default initial positions for myactuator_arm's ros2_control fake system

initial_positions:
  joint1: 0.0
  joint2: 0.0
  joint3: 0.0
  joint4: 0.0
  joint5: 0.0